source/features/convert-release-to-draft.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import LoadingIcon from '../github-helpers/icon-loading';
+import {getRepoURL} from '../github-helpers';
+
+async function convertToDraft({delegateTarget: draftButton}: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	try {
+		await api.expectToken();
```

Comment:
This should be in `init`

Reply:
Agree but over here I wanted to catch both, api and return errors. 

I should do !apikey return false?